### PR TITLE
Update type definition and README.md with isPrime's optional strict parameter

### PR DESCRIPTION
**Pull request by silentsilas**
I'm building an interactive RSA tool, and I just about finished the key generation for up to 4086-bit keys when I realized Safari has yet to implement BigInt. So, needless to say, I wished to thank you for this library/polyfill and contribute what little I can to it.

Also, is there an ELI5 for that strict parameter? I understand it changes how many integers it tests against, but I haven't a clue what GRH is, lol.

### CODE REVIEW

**Comment by gardhr** [comment]
Basically, isPrime(true) gives you a 100% certificate of primality based on proven mathematics whereas the much more efficient isPrime(false) relies on the unproven yet widely-accepted-as-true [GRH](https://en.m.wikipedia.org/wiki/Generalized_Riemann_hypothesis).  

Incidentally, invoking isProbablePrime with the default parameters is already tells you with something like 99.9% certainty whether or not a number is prime (and in a fraction of the time required for isPrime) so unless you just need cryptographically secure primes or what have you then isProbablePrime is really the best choice. ;)

**Comment by Poeticode** [comment]
Ah, that makes sense! We must have skimmed over GRH in my Intro to Crypto course. Thank you.

**Comment by peterolson** [comment]
Sorry for the long delay in merging, I must have overlooked it before.
